config/locales/server.en.yml

```diff
@@ -2215,6 +2215,7 @@ en:
     group_in_subject: "Set %%{optional_pm} in email subject to name of first group in PM, see: <a href='https://meta.discourse.org/t/customize-specific-email-templates/88323' target='_blank'>Customize subject format for standard emails</a>"
     allow_anonymous_posting: "Allow users to switch to anonymous mode"
     allow_anonymous_likes: "Allow anonymous users to like posts"
+    allow_anonymous_to_inherit_group: "Have anonymous users inherit their user's groups if enabled"
```

Comment:
I could add a warning here as per [Matt's comment](https://meta.discourse.org/t/pr-review-allow-anonymous-users-to-inherit-their-parent-user-groups/292692/2?u=meltingmettle_gh).

> Warning - enabling this feature may make anonymous users identifiable if their combination of group memberships is sufficiently unique